apps/ledger-live-mobile/e2e/setup.ts

```diff
@@ -4,6 +4,11 @@ import * as bridge from "./bridge/server";
 
 beforeAll(async () => {
   bridge.init();
+
+  // Certain setups might need the following 2 lines to work in debug mode
+  await device.reverseTcpPort(8081);
+  await device.reverseTcpPort(8099);
```

Comment:
What process is running on `8099`?

Reply:
It's the websocket service that communicates between the Detox server and the App itself.